components/homme/src/share/global_norms_mod.F90

```diff
@@ -853,8 +862,19 @@ subroutine wrap_repro_sum (nvars, comm, nsize)
     do n=1,nvars
        do i=1,nsize_use
           if (global_shared_buf(i,n) /= global_shared_buf(i,n) ) then
-               print *, "var,nvars:",n,nvars
+
```

Comment:
Do you mean for this code to go in? Seems like it's from a different PR.

Reply:
this will be removed before going in, i might still need it here.